src/providers/definition/controllerDefinitionProvider.ts

```diff
@@ -12,6 +12,12 @@
 				return [ path.join(project.filePath, 'app', 'lib', `${value}.js`) ];
 			}
```

Comment:
<!--
  1 failure:  src/providers/def...
  2 warning:  src/providers/def..., src/providers/def...
  
  
  DangerID: danger-id-Danger;
  File: src/providers/definition/controllerDefinitionProvider.ts;
  Line: 18;
-->
- :no_entry_sign: src/providers/definition/controllerDefinitionProvider.ts line 18 – Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
- :warning: src/providers/definition/controllerDefinitionProvider.ts line 18 – A space is required after '['. (array-bracket-spacing)
- :warning: src/providers/definition/controllerDefinitionProvider.ts line 18 – A space is required before ']'. (array-bracket-spacing)